sources/effects.c

```diff
@@ -242,6 +244,27 @@ void toggle_tool_select(int fd_touch) {
     activate_tool_select(fd_touch);
 }
 
+void one_off_erase_select(int fd_touch) {
+  activate_tool_eraser_select(fd_touch);
+  oneOffEraseSelect = true;
+}
+
+void one_off_eraser(int fd_pen) {
+  activate_tool_eraser(fd_pen);
+  oneOffEraser = true;
+}
+
+void pen_up(int fd_touch, int fd_pen) {
+  if (oneOffEraseSelect) {
+    deactivate_tool_eraser_select(fd_touch);
+    oneOffEraseSelect = false;
+  }
```

Comment:
I haven't tested the `one-off-eraser-selection` effect on Remarkable version 3.8, so I'm not sure if this will work for you -- on version 3.9 it doesn't work at all, I have to add a delay (see [here](https://github.com/rish987/RemarkableLamyEraser/blob/f51b51cc6d9070a874b369ae9f6fbbeafafa9ea7/sources/effects.c#L293)).

Reply:
Okay, having recently downgraded to 3.8 I can confirm that you do need a delay. I'm AFC for the next few days but [here is my personal development branch](https://github.com/rish987/RemarkableLamyEraser/tree/hl-toggle-long-click) if you're curious.